ngraph/src/ngraph/op/proposal.cpp

```diff
@@ -121,3 +121,25 @@ bool AttributeAdapter<op::ProposalAttrs>::visit_attributes(AttributeVisitor& vis
     visitor.on_attribute("framework", m_ref.framework);
     return true;
 }
+
+constexpr NodeTypeInfo op::v4::Proposal::type_info;
+
+op::v4::Proposal::Proposal(const Output<Node>& class_probs,
+                           const Output<Node>& class_bbox_deltas,
+                           const Output<Node>& image_shape,
+                           const op::ProposalAttrs& attrs)
+    : v0::Proposal(class_probs, class_bbox_deltas, image_shape, attrs)
+{
+    constructor_validate_and_infer_types();
+}
+
+void op::v4::Proposal::validate_and_infer_types()
+{
+    v0::Proposal::validate_and_infer_types();
+}
+
+std::shared_ptr<Node> op::v4::Proposal::clone_with_new_inputs(const OutputVector& new_args) const
+{
+    check_new_args_count(this, new_args);
+    return make_shared<op::v4::Proposal>(new_args.at(0), new_args.at(1), new_args.at(2), m_attrs);
+}
```

Comment:
I don't see any difference between v0 and v4 operations...

Reply:
Now they differ. v0 have 1 output, v4 has 2 outs and `proposal_ie` have either 1 or 2 depending on the node which he replaces. Please review changes.